src/shared/game/interfaces.ts

```diff
@@ -75,8 +75,8 @@ export interface IAdditionalApplicationInfo {
    * (If false, this will only run when specifically launched)
    */
   autoRunBefore: boolean;
-  /** Command line argument(s) used when launching the additional application */
-  commandLine: string;
+  /** Command line argument(s) passed to the application to launch the game */
+  launchCommand: string;
```

Comment:
I think these properties should have similar names to their XML counterparts

Reply:
We already refer to it as a Launch Command everywhere seen in the launcher ui, just not the code base, and we already refer to Game CommandLine as launchCommand in the code base too. I'd disagree.

Reply:
I found 3 places where it was referred to as "launch command", and only one of them was visible to the end user. But I don't mind it too much so keep it like this if you want. I also noticed that there are some leftover language strings ("commandLine" and "noCommandLine")